app/main.py

```diff
@@ -1,2 +1,86 @@
+from typing import Optional, Any
+
+
+class Node:
+    def __init__(self, key: Any, hash_value: int, value: Any) -> None:
```

Comment:
Place all arguments on a new line to match the code style for the `__init__` of the `Dictionary` class.